### DISCUSSION TITLE
Drag and Drop

### DISCUSSION THREAD

**Discussion by PSanilP** [vkurko/calendar#121] [General]
Is there a demo for the callback on dragging and dropping the event to another date?
I am familiar with dragula which has a feature like
`<html>
<body>

drake.on("drop", function (el, target, source, sibling) {
--
  |  
  | //console.log(source.id);
  | //console.log(el.id);
  | //console.log(target.id);
.. so on

</body>
</html>`

**Comment by vkurko** [comment]
I've prepared a [demo](https://jsbin.com/jemenatato/edit?js,console,output) for you.
